### PR TITLE
Add a elasticsearch commands to control replicas

### PR DESCRIPTION
**Pull request by esoergel**
@nickpell

### CODE REVIEW

File: scripts/elasticsearch-administer.py [hunk 1]
```diff
@@ -56,6 +56,23 @@ def cluster_settings(es):
...
+
+def create_replica_shards(es):
+    # https://www.elastic.co/guide/en/elasticsearch/reference/2.3/indices-update-settings.html
+    indices = IndicesClient(es)
+    pprint(indices.put_settings({"index.number_of_replicas" : 1}, "_all"))
+
+
+def cancel_replica_shards(es):
+    indices = IndicesClient(es)
+    pprint(indices.put_settings({"index.number_of_replicas" : 0}, "_all"))
```
Comment (dimagimon) [inline scripts/elasticsearch-administer.py]: E203 whitespace before ':'

File: scripts/elasticsearch-administer.py [hunk 2]
```diff
@@ -56,6 +56,23 @@ def cluster_settings(es):
...
 
+def index_settings(es):
+    indices = IndicesClient(es)
+    pprint(indices.get_settings(flat_settings=True))
+
+
+def create_replica_shards(es):
+    # https://www.elastic.co/guide/en/elasticsearch/reference/2.3/indices-update-settings.html
+    indices = IndicesClient(es)
+    pprint(indices.put_settings({"index.number_of_replicas" : 1}, "_all"))
```
Comment (dimagimon) [inline scripts/elasticsearch-administer.py]: E203 whitespace before ':'

**Comment by nickpell** [comment]
I probably don't have to context to merge, but looks fine to me

**Comment by esoergel** [comment]
@dannyroberts is probably more familiar

**Comment by dannyroberts** [comment]
👍

**Pull request merged**